### PR TITLE
Changed source code to satisfy tslint

### PR DESCRIPTION
**Pull request by teobugslayer**

### CODE REVIEW

**Comment by justcodebuilduser** [comment]
:heart:

File: child-process.ts [hunk 1]
```diff
@@ -82,9 +82,9 @@ export class ChildProcess implements IChildProcess {
 				if (exitCode === 0) {
 					future.return(result);
 				} else {
-					let errorMessage = util.format('Command %s failed with exit code %d', command, exitCode);
+					let errorMessage = util.format("Command %s failed with exit code %d", command, exitCode);
```
Comment (Mitko-Kerezov) [inline child-process.ts]: This message and the one below can be rewritten as template strings

File: commands/device/list-devices.ts [hunk 2]
```diff
@@ -19,18 +17,20 @@ export class ListDevicesCommand implements ICommand {
...
-					}))}).future<void>()();
+					return (() => { 
+						this.$logger.out(JSON.stringify({
+							identifier: device.deviceInfo.identifier,
+							platform: device.deviceInfo.platform,
+							model: device.deviceInfo.model,
+							name: device.deviceInfo.displayName,
+							version: device.deviceInfo.version,
+							vendor: device.deviceInfo.vendor
+						}));
```
Comment (Fatme) [inline commands/device/list-devices.ts]: I think we can simplify this code. Just `this.$logger.out(JSON.stringify(device.deviceInfo))` :smile_cat:

**Comment by justcodebuilduser** [comment]
:heart:

**Comment by Mitko-Kerezov** [comment]
:+1: as far as I can see

**Pull request merged**